services/IoT/applications/libraries/python.md

```diff
@@ -661,7 +656,7 @@ The following code sample outlines the format that is required for this method:
 
 Use the *getDevice()* method to remove a device from an organization on the {{site.data.keyword.iot_short_notm}}. When you retrieve device details using the *getDevice()* method, you must specify the deviceTypeId and deviceId parameters
```

Comment:
"Use the *getDevice()* method to remove a device from an organization on the {{site.data.keyword.iot_short_notm}}."
This does not sound right - getting the device details removes the device? Or you can use the details that you get to remove the revice? Clarify.